config/pmd.xml

```diff
@@ -316,7 +316,8 @@
       <!-- Main has an annotated field for each command line option. This is by design. -->
       <property name="violationSuppressXPath"
                 value="//ClassOrInterfaceDeclaration[@SimpleName='Checker']
-    | //ClassOrInterfaceDeclaration[@SimpleName='Main']"/>
+    | //ClassOrInterfaceDeclaration[@SimpleName='Main']
+    | //ClassOrInterfaceDeclaration[@SimpleName='ImportOrderCheck']"/>
```

Comment:
Suppress this because PWD has too many class fields.

Reply:
I am ok with this.